cfep-09.md

```diff
@@ -0,0 +1,100 @@
+
+### The rule of preferring the most recent pinnings file covers all of the relevant cases
+ - If a feedstock has no local pinnings, then that is fine. 
+ - If the file is present, but matches the global pinnings, this is also fine.
+ - In the middle of a migration, a feedstock that is being migrated would have a local pinnings file that should be used ahead of the global one. The local version would be ahead of global in this case since the migration bot would add the new pinnings file as the local one.
+ - After a migration has ended and the changes from the migration are merged to the global pinnings file, then the global file will always be at the same version as the local one, or at a later version.
+- If some other migration comes through later, it would then overwrite the same local pinnings file with a version that is by definition later than the current global one. 
+- If a feedstock adds a dependency that has a pinned version, then the version that gets used is from the most recent pinnings file between the global one and the local one. 
+- Adding a dependence in the middle of a migration would cause the repo to be detected by the bot and a migration PR would be issued.
+
+Thus we don't need to have conda-smithy cleanup any old local pinnings files, though it could if we want.
+
+
+## Backwards Compatibility
+
+This proposal would require that all changes to the global pinnings file have to be routed through the new `migrations` repo. No human would make a PR on the global pinnings repo and instead that repo would be managed entirely by the bot. 
+
+## Alternatives
+
+1. Earlier versions of this proposal did not direct version the global pinnings files in a time ordered fashion. Instead they simply always preferred the local pinnings over the global ones. However, this procedure fails in the case of a packahe adding a new dependence with a pin and it having an outdated local pinnings file. By having a version stored in the file, `conda-smithy` can tell which version is more recent.
+
+2. An earlier version of this propsal was designed around a migration being a YAML snippet. However, `conda_build_config.yaml` files can have conda-build selectors in them (e.g., `# [osx]`). This conda-build feature makes it pretty hard to use a YAML snippet as a patch on th global pinnings file. Using a direct copy of the proposed new global pinnings file is simpler. This procedure also allows us to migrate almost anything in the file. 
+
+3. Some suggestions have been centered around directly looking at the package versions in the pinnings file to determine which one is newer. This procedure has some flaws. First, it does not allow us to downgrade pinnings if needed. It is ambiguous in the case that we migrate more than one package where one version goes up and the other goes down. Finally, it does not cover changes to things like selectors or zip keys which don't have version associated with them. 
```

Comment:
Typo/grammar-o: "don't have version associated with them" should probably have "versions"?